src/engraving/libmscore/rest.cpp

```diff
@@ -188,7 +187,7 @@ bool Rest::acceptDrop(EditData& data) const
         ) {
         return true;
     }
-    return false;
+    return type != ElementType::SLUR && e->isSpanner();
```

Comment:
for now, keep existing behaviour that we don't allow dropping slurs on rests, even though it actually works fine...